File: C programming/Assignment/Assignment3/table.c
#include <stdio.h>
void main(){
    int num,res,i = 1;
    printf("enter the number:");
    scanf("%d",&num);
    while(i<=10){
        res = num * i;
        printf("%d * %d = %d\n",num,i,res);
        i++;
    }
    
}

File: Java_programming/basic_java/assignment3/table.java
package assignment3;

public class table {

	public static void main(String[] args) {
		int num=5,res,i = 1;
//	    printf("enter the number:");
//	    scanf("%d",&num);
	    while(i<=10){
	        res = num * i;
	        System.out.printf("%d * %d = %d\n",num,i,res);
	        i++;
	    }
	}

}
